web/src/main/ui/app/components/flows-new/edit-flow/custom/ui/custom-ui.component.ts

```diff
@@ -11,16 +12,10 @@ export class CustomUiComponent {
   @Input() module: string;
   @Output() updateCustom = new EventEmitter();
 
-  constructor() {
-  }
-
-  onKeyChange(event) {
-    if (event.key === 'Enter') {
-      this.onChange();
-    }
+  constructor(private snackBar: MatSnackBar) {
   }
 
   onChange() {
     this.updateCustom.emit(this.step);
-  }
+    this.snackBar.open("Change Saved", "", {panelClass: ['snackbar'], duration: 1200});  }
```

Comment:
the snackbar open code should be in the edit-flow.component.ts where @briantang originally had it. This way the snackbar doesn't appear until we get the response from the server.